docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -296,6 +297,52 @@ To enable denormals optimization in the application, the `denormals_optimization
 
+### Sparse weights decompression
+`Sparse weights` are weights where most of the elements are zero. The ratio of the number of zero elements to the number of all elements is called `sparse rate`. Thus, we assume that `sparse weights` are weights with a high sparse rate. In case of `sparse weights`, we can store only non-zero values in memory using special storage structures, which allows us to use memory more efficiently. In turn, this can give us better performance in the high memory bound workloads (e.g., throughput scenario).
+
+`Sparse weights decompression feature` allows to pack weights for Matrix Multiplication operations directly in the CPU plugin at the model compilation stage and store non-zero values in a special packed format. Then, during the execution of the model, the weights are unpacked and used in the computational kernel.
+
+To use this feature, the user is provided with property `sparse_weights_decompression_rate`, which can take values from the interval [0, 1]. `sparse_weights_decompression_rate` is the minimum fraction of nulls above which we use weights compression. The default value is `1`, which means the option is disabled.
+
+> **NOTE**: **`Sparse weights decompression feature` is disabled by default** since overall speedup highly depends on particular workload and for some cases the feature may introduce performance degradations.
+
+If the user passes an option with a value other than `1`, then during the compilation of the model for all weights of the Matrix Multiplication operations, the number of zero elements (sparse_rate) will be calculated. If sparse_rate is higher than `sparse_weights_decompression_rate` then the weights will be packed into a special compressed format (packed).
+
+> **Example**: If user set `sparse_weights_decompression_rate` to `0.8`, then for a tensor with a calculated sparse rate of `0.9`, the weights will be packed into a special format and the weights will be decompressed at the kernel execution stage. For a tensor with a sparse rate of `0.7`, the weights will remain in memory in a dense format.
+
+Code examples how to use `sparse_weights_decompression_rate`:
+
+@sphinxdirective
+
+.. tab:: C++
+
+      .. doxygensnippet:: docs/snippets/cpu/ov_sparse_weights_decompression.cpp
+         :language: cpp
+         :fragment: [ov:intel_cpu:sparse_weights_decompression:part0]
+
+.. tab:: Python
+
+      .. doxygensnippet:: docs/snippets/cpu/ov_sparse_weights_decompression.py
+         :language: python
+         :fragment: [ov:intel_cpu:sparse_weights_decompression:part0]
+
+@endsphinxdirective
+
+> **NOTE**: The `sparse_weights_decompression_rate` property must be set before calling `compile_model()`.
+
+Information about the layers in which the `sparse weights decompression featur`e was applied can be obtained from perf counters log. The "exec type" field will contain the implementation type with the "**sparse**" particle ("**brgemm_avx512_amx_sparse_I8**" in the example below):
```

Comment:
`sparse weights decompression featur`e - should be fixed

Reply:
fixed, thanks